### PR TITLE
Update GCP doc and scripts [skip ci]

### PR DESCRIPTION
**Pull request by viadea**
Updated the GCP doc and scripts to reflect the most recent GCP init scripts to address https://github.com/NVIDIA/spark-rapids/issues/5282 . 

1. custom image script was out of date. Updated it based on latest image version and latest GCP init script. [I have tested the new custom image script and it worked fine.]
2. changed "key value" to "key=value" for Dataproc creation scripts for every place to match GCP doc.
3. Added some notes in notebook to mention how to download the input file.
4. Some rewording based on @tgravescs 's drafted [PR](https://github.com/NVIDIA/spark-rapids/pull/5283)

### CODE REVIEW

**Comment by nvliyuan** [comment]
verified “## Submit Spark jobs to a Dataproc Cluster Accelerated by GPUs” section, works fine

File: docs/get-started/getting-started-gcp.md [hunk 1]
```diff
@@ -89,8 +93,11 @@ If you'd like to further accelerate init time to 4-5 minutes, create a custom Da
 To use notebooks with a Dataproc cluster, click on the cluster name under the Dataproc cluster tab
 and navigate to the "Web Interfaces" tab.  Under "Web Interfaces", click on the JupyterLab or
 Jupyter link to start to use sample 
-[Mortgage ETL on GPU Jupyter Notebook](../demo/GCP/Mortgage-ETL-GPU.ipynb) to process full 17 years 
-[Mortgage data](https://docs.rapids.ai/datasets/mortgage-data).
+[Mortgage ETL on GPU Jupyter Notebook](../demo/GCP/Mortgage-ETL-GPU.ipynb).
```
Comment (tgravescs) [inline docs/get-started/getting-started-gcp.md:96]: how does hte ipynb file get to the notebook, is there an easy way to open it from the notebook directly or do you have to download and then reupload?

File: docs/get-started/getting-started-gcp.md [hunk 2]
```diff
@@ -89,8 +93,11 @@ If you'd like to further accelerate init time to 4-5 minutes, create a custom Da
...
+
+To get the input data of the sample notebook, we need to download the full 17 years' [Mortgage 
+data](https://docs.rapids.ai/datasets/mortgage-data) as `mortgage_2000-2016.tgz`, uncompress and 
+upload it to a GCS bucket. Above notebook has handy commands to do this step on master node.
```
Comment (tgravescs) [inline docs/get-started/getting-started-gcp.md:100]: do we want to put anything about how to upload?  perhaps just a link to gcs docs

Comment (viadea) [reply docs/get-started/getting-started-gcp.md:100]: @tgravescs The above notebook has detailed steps on how to download, uncompress and upload. So I thought we do not duplicate it in the doc here?

File: docs/get-started/getting-started-gcp.md (hunk 1 above)
Comment (viadea) [reply docs/get-started/getting-started-gcp.md]: As per my test, only upload/download option. So I just added some words to let user firstly download and upload it in jupyter.

File: docs/get-started/getting-started-gcp.md [hunk 3]
```diff
@@ -129,17 +136,18 @@ val (xgbClassificationModel, _) = benchmark("train") {
...
 ## Submit Spark jobs to a Dataproc Cluster Accelerated by GPUs
-Similar to spark-submit for on-prem clusters, Dataproc supports a Spark applicaton job to be
+Similar to spark-submit for on-prem clusters, Dataproc supports a Spark application job to be
 submitted as a Dataproc job.  The mortgage examples we use above are also available as a [spark
-application](https://github.com/NVIDIA/spark-xgboost-examples/tree/spark-3/examples/apps/scala).
+application](https://github.com/NVIDIA/spark-rapids-examples/tree/branch-22.06/examples/Spark-ETL%2BXGBoost).
 After [building the jar
-files](https://github.com/NVIDIA/spark-xgboost-examples/blob/spark-3/getting-started-guides/building-sample-apps/scala.md)
-they are available through maven `mvn package -Dcuda.classifier=cuda11-0`. 
+files](https://github.com/NVIDIA/spark-rapids-examples/blob/branch-22.06/docs/get-started/xgboost-examples/building-sample-apps/scala.md)
```
[the remark this answers is not in the document]
Comment (viadea) [reply docs/get-started/getting-started-gcp.md]: Make sense. just updated it.

File: docs/get-started/gpu_dataproc_packages_ubuntu_sample.sh [hunk 4]
```diff
@@ -20,7 +20,7 @@ OS_NAME=$(lsb_release -is | tr '[:upper:]' '[:lower:]')
 readonly OS_NAME
 OS_DIST=$(lsb_release -cs)
 readonly OS_DIST
-CUDA_VERSION='11.0'
+CUDA_VERSION='11.2'
```
[the remark this answers is not in the document]
Comment (viadea) [reply docs/get-started/gpu_dataproc_packages_ubuntu_sample.sh]: Changed it back to 11.0.

File: docs/get-started/getting-started-gcp.md [hunk 5]
```diff
@@ -63,17 +67,17 @@ The script below will initialize with the following:
...
+    --num-workers=$NUM_WORKERS \
+    --worker-accelerator=type=nvidia-tesla-t4,count=$NUM_GPUS \
+    --worker-machine-type=n1-highmem-32\
+    --num-worker-local-ssds=4 \
+    --initialization-actions=gs://goog-dataproc-initialization-actions-${REGION}/gpu/install_gpu_driver.sh,gs://goog-dataproc-initialization-actions-${REGION}/rapids/rapids.sh \
     --optional-components=JUPYTER,ZEPPELIN \
     --metadata rapids-runtime=SPARK \
-    --bucket $GCS_BUCKET \
+    --bucket=$GCS_BUCKET \
     --enable-component-gateway
```
[the remark this answers is not in the document]
Comment (viadea) [reply docs/get-started/getting-started-gcp.md]: Yes i added this based on our current network setting.

File: docs/get-started/getting-started-gcp.md [hunk 6]
```diff
@@ -129,17 +136,18 @@ val (xgbClassificationModel, _) = benchmark("train") {
 ```
 
 ## Submit Spark jobs to a Dataproc Cluster Accelerated by GPUs
-Similar to spark-submit for on-prem clusters, Dataproc supports a Spark applicaton job to be
+Similar to spark-submit for on-prem clusters, Dataproc supports a Spark application job to be
 submitted as a Dataproc job.  The mortgage examples we use above are also available as a [spark
-application](https://github.com/NVIDIA/spark-xgboost-examples/tree/spark-3/examples/apps/scala).
+application](https://github.com/NVIDIA/spark-rapids-examples/tree/branch-22.06/examples/Spark-ETL%2BXGBoost).
```
[the remark this answers is not in the document]
Comment (viadea) [reply docs/get-started/getting-started-gcp.md]: done

File: docs/get-started/getting-started-gcp.md [hunk 7]
```diff
@@ -129,17 +136,18 @@ val (xgbClassificationModel, _) = benchmark("train") {
...
-files](https://github.com/NVIDIA/spark-xgboost-examples/blob/spark-3/getting-started-guides/building-sample-apps/scala.md)
-they are available through maven `mvn package -Dcuda.classifier=cuda11-0`. 
+files](https://github.com/NVIDIA/spark-rapids-examples/blob/branch-22.06/docs/get-started/xgboost-examples/building-sample-apps/scala.md)
+.
 
-Place the jar file `sample_xgboost_apps-0.2.2.jar` under the `gs://$GCS_BUCKET/scala/` folder by
-running `gsutil cp target/sample_xgboost_apps-0.2.2.jar gs://$GCS_BUCKET/scala/`.  To do this you
-can either drag and drop files from your local machine into the GCP storage browser, or use the
-gsutil cp as shown before to do this from a command line.  We can thereby submit the jar by:
+Place the jar file `sample_xgboost_apps-0.2.2-jar-with-dependencies.jar` under the
```
[the remark this answers is not in the document]
Comment (viadea) [reply docs/get-started/getting-started-gcp.md]: Changed to <version>

**Review by nvliyuan — APPROVED** [review]

**Review by tgravescs — APPROVED** [review]

**Pull request merged**